syz-manager/manager.go

```diff
@@ -1425,18 +1435,10 @@ func (mgr *Manager) minimizeCorpusLocked() {
 		log.Logf(0, "coverage for %v has saturated, not accepting more inputs", call)
 	}
 
-	// Don't minimize persistent corpus until fuzzers have triaged all inputs from it.
-	if mgr.phase < phaseTriagedCorpus {
-		return
-	}
 	mgr.corpusDBMu.Lock()
 	defer mgr.corpusDBMu.Unlock()
-	for key := range mgr.corpusDB.Records {
-		ok1 := mgr.corpus.Item(key) != nil
-		_, ok2 := mgr.disabledHashes[key]
-		if !ok1 && !ok2 {
-			mgr.corpusDB.Delete(key)
-		}
+	for inp := range dropped {
```

Comment:
Will the new approach drop the corpus items that have been discarded during the triage as the old one used to do?

Reply:
Good point.
I've switched to a different approach (we only discard records values), which is I think simpler, less pervasive, and more obviously equivalent to the current behavior
PTAL.